cmd/swctl/cmd_status.go

```diff
@@ -34,23 +64,240 @@ func NewStatusCmd(cli Cli) *cobra.Command {
 			UnknownFlags: true,
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			opts.Args = args
 			return runStatusCmd(cli, opts)
 		},
 	}
+	opts.InstallFlags(cmd.PersistentFlags())
 	return cmd
 }
 
-func runStatusCmd(cli Cli, opts StatusCmdOptions) error {
-	cmd := fmt.Sprintf("vpp-probe --env=%s discover", defaultVppProbeEnv)
-	out, err := cli.Exec(cmd, opts.Args)
-	if err != nil {
-		if ee, ok := err.(*exec.ExitError); ok {
-			return fmt.Errorf("%v: %s", ee.String(), ee.Stderr)
+type statusInfo struct {
+	client.Component
+	ConfigCounts configCounts
+}
+
+func runStatusCmd(cli Cli, opts StatusOptions) error {
+	if opts.ShowInterfaces {
+		cmd := fmt.Sprintf("vpp-probe --env=%s --query label=%s=stonework discover", defaultVppProbeEnv, client.DockerComposeServiceLabel)
```

Comment:
The query should not be limited to stonework service only. It should run for all components.. if the query will fail with non-vpp contianers, then run it for all components that have VPP. 

For example our setup for MSR would not show the interfaces in vswitch, which is vanilla vpp-agent.